### ISSUE TITLE
Solution in 02_Ten_../04_loss_.., sigmoid entropy loss don't work well

### ISSUE THREAD

**Issue by leigaoyi** [nfmcclure/tensorflow_cookbook#25]
tf.nn.sigmoid_cross_entropy_with_logits(_sentinel=None, labels=None, logits=None, name=None) don't suggest using _sentinel , because it refer to positional parameters.
But in my computer and python environment, if just put in
**tf.nn.sigmoid_cross_entropy_with_logits(x_vals,targets)**
It gives warnings, so I think maybe this function receive x_vals as _sentinel. When add the labels , I get the resule:
**xentropy_sigmoid_y_vals = tf.nn.sigmoid_cross_entropy_with_logits(labels=x_vals,logits= targets)**
This can work on python3.5 and tensorflow 1.0.0

**Comment by nfmcclure** [comment]
Hi @leigaoyi , this should be fixed now.  I also added a jupyter notebook in the section as well.  Let me know if you have any other issues.  Thanks!
